src/rsa/signing.rs

```diff
@@ -256,16 +392,13 @@ struct BN_BLINDING {
 extern {
     fn GFp_BN_BLINDING_new() -> *mut BN_BLINDING;
     fn GFp_BN_BLINDING_free(b: *mut BN_BLINDING);
-    fn GFp_rsa_new_end(rsa: *mut RSA, n: &bigint::BIGNUM, d: &bigint::BIGNUM,
-                       p: &bigint::BIGNUM, q: &bigint::BIGNUM,
-                       iqmp: &bigint::BIGNUM) -> c::int;
 }
 
 #[allow(improper_ctypes)]
 extern {
-    fn GFp_rsa_private_transform(rsa: *const RSA, inout: *mut u8,
-                                 len: c::size_t, blinding: *mut BN_BLINDING,
-                                 rng: *mut rand::RAND) -> c::int;
+    fn GFp_rsa_private_transform(rsa: &RSA, base: &mut bigint::BIGNUM,
```

Comment:
`&mut` means “inout,” right? AFAICT `/*inout*/` only appears in C code for the declarations of *this* function.